src/proto/metadata.proto

```diff
@@ -0,0 +1,160 @@
+syntax = "proto3";
```

Comment:
This file is engine internal and not needed in the client :) 
Please delete it!

Reply:
Done, thx!